consensus/istanbul/validator/weighted.go

```diff
@@ -553,6 +572,36 @@ func (valSet *weightedCouncil) F() int {
 
 func (valSet *weightedCouncil) Policy() istanbul.ProposerPolicy { return valSet.policy }
 
+func (valSet *weightedCouncil) Update(hash common.Hash, blockNum uint64) error {
+	valSet.validatorMu.Lock()
+	defer valSet.validatorMu.Unlock()
+
+	newStakingInfo := reward.GetStakingInfo(blockNum + 1)
+
+	valSet.stakingInfo = newStakingInfo
+	if valSet.stakingInfo == nil {
+		// Just return without updating validators
+		return errors.New("skip updating validators due to no staking info")
+	}
```

Comment:
```suggestion
	newStakingInfo := reward.GetStakingInfo(blockNum + 1)
	if newStakingInfo == nil {
		// Just return without updating validators
		return errors.New("skip updating validators due to no staking info")
	}
	valSet.stakingInfo = newStakingInfo
```
If you don't mind, could you change it? I think it's more intuitive.

Reply:
This is from other code fragments and updated it as well.